src/core/adapter.cpp

```diff
@@ -21,19 +21,20 @@ USE_STRING_ID(ANY_ADAPTER);
 #include "ecet.h"
 
 CAdapter::CAdapter(forte::core::CFBContainer &paContainer, const SFBInterfaceSpec& paInterfaceSpecSocket, const CStringDictionary::TStringId paInstanceNameId, const SFBInterfaceSpec& paInterfaceSpecPlug, bool paIsPlug) :
-  CGenFunctionBlock<CFunctionBlock>(paContainer, (paIsPlug) ? paInterfaceSpecPlug : paInterfaceSpecSocket, paInstanceNameId),
-  mParentAdapterListEventID(0),
-  mIsPlug(paIsPlug),
-  mPeer(nullptr),
-  mLocalDIs(mDIs),
-  mAdapterConn(nullptr){
+    CGenFunctionBlock<CFunctionBlock>(paContainer, (paIsPlug) ? paInterfaceSpecPlug : paInterfaceSpecSocket, paInstanceNameId),
```

Comment:
I would suggest to remove the parantheses around paIsPlug, as it makes the code harder to read

Reply:
done